### PR TITLE
Max's review work

### PR DESCRIPTION
**Pull request by maxckelly**
Max's Work

### CODE REVIEW

File: ruby_review_week_four.rb [hunk 1]
```diff
@@ -2,36 +2,134 @@
 
 # 1. What is a variable?
 
+=begin
+  
+A Variable is a element where you can store and access a string, integer, boolean. Variables store a single value e.g.
```
Comment (harrisonmalone) [inline ruby_review_week_four.rb:7]: great answer, you could also mention how variables are reusable

File: ruby_review_week_four.rb [hunk 2]
```diff
@@ -2,36 +2,134 @@
...
+#   puts arr
+# end
+
 # 3. In Ruby, what are booleans?
 
+# Booleans are true or false values e.g. var = true, var = false
+
 # 4. How do classes relate to objects?
 
+# Classes are like the template (structure) of objects. They create the basic overview of what each object will contain.
```
Comment (harrisonmalone) [inline ruby_review_week_four.rb:26]: great answer!

File: ruby_review_week_four.rb [hunk 3]
```diff
@@ -2,36 +2,134 @@
...
+
 # 5. What is an instance variable?
 
+# An instance variable is a variable that is initizled through the creation of a class e.g.
```
Comment (harrisonmalone) [inline ruby_review_week_four.rb:30]: this is true, an answer i'd like would include the word `attribute`

> an instance variable is an attribute of an object

File: ruby_review_week_four.rb [hunk 4]
```diff
@@ -2,36 +2,134 @@
...
+
 # 6. Choose an everyday item (cup, computer, person) and give it two instance variables (which will be attributes), and one method (to describe a function of the object). Code the initialize method in full (for these attributes only), and the definition line of the instance method (but do not code the entire method).
 
+class Person
+  def initialize(name, age)
+    @name = name
+    @age = age
+  end
+
+  def to_string
```
Comment (harrisonmalone) [inline ruby_review_week_four.rb:47]: make sure you follow instructions precisely, no need to code this entire method in full

File: ruby_review_week_four.rb [hunk 5]
```diff
@@ -2,36 +2,134 @@
...
+  
+=end
+# 9. How would you use that class to make three fish objects?
+  # fish_one = Fish.new
+  # fish_two = Fish.new
+  # fish_three = Fish.new
 # 10. Add two attributes to your fish class, name and species, and make them accessible for reading and writing.
 
+class Fish
+  attr_accessor :name, :species
```
Comment (harrisonmalone) [inline ruby_review_week_four.rb:74]: ✅

File: ruby_review_week_four.rb [hunk 6]
```diff
@@ -2,36 +2,134 @@
...
+
 # 11. 
 
 # a. Create one fish object and store it in a variable, and 
 
+fish_one = Fish.new("Nemo", "Clown Fish")
+
 # b. on a second line, give that fish object’s name attribute the value of ‘Fred’ (assume you have access).
 
+fish_one.name = "Fred"
```
Comment (harrisonmalone) [inline ruby_review_week_four.rb:91]: perfect!!!

File: ruby_review_week_four.rb [hunk 7]
```diff
@@ -2,36 +2,134 @@
...
 
+student = {
+  name: "Max",
+  age: 21
+}
+
 # 13. Write a Ruby method that takes three arguments and returns the sum (total) of those three arguments.
 
+def ruby_method(x,y,z)
+  return x + y + z
```
Comment (harrisonmalone) [inline ruby_review_week_four.rb:104]: nice

File: ruby_review_week_four.rb [hunk 8]
```diff
@@ -2,36 +2,134 @@
...
+
+puts ruby_method(20,30,40)
+
 # 14. Write a Ruby method that takes an array and a string as arguments, and returns true if the string is in the array, and false if it is not.
 
+# array = [1]
+# def ruby_method_two(array, string)
+
+#   array.each do |arr| 
+#     if arr.is_a?(String)
```
Comment (harrisonmalone) [inline ruby_review_week_four.rb:115]: this has a few issues, we'll review it in class next week

File: ruby_review_week_four.rb [hunk 9]
```diff
@@ -2,36 +2,134 @@
...
+
+# puts ruby_method_two(array, 2)
+
 # 15. Write a method that takes one argument, a number, and that returns true if the number is divisible by three, and false otherwise. 
 
-# 16. You must use your previous method in this next question. Write another method that takes an array (of numbers) as an argument. This method will use your first method to return two arrays (within one array - an array of arrays). The first array contains the numbers that are divisible by 3, and the other contains the rest of the numbers. For example, if you pass the second method the array [9, 4, 3, 5, 30, 2, 8] it would return [[9, 3, 30],[4, 5, 2, 8]].
\ No newline at end of file
+numbers = [9, 4, 3, 5, 30, 2, 8]
+
+def ruby_method_three(number_array)
```
Comment (harrisonmalone) [inline ruby_review_week_four.rb:129]: we'll review these in class next week

File: ruby_review_week_four.rb (hunk 8 above)
Comment (maxckelly) [reply ruby_review_week_four.rb:115]: No worries! I think I over complicated it and thought that the question was asking if the array is changed it should pick up the difference but chat next week about it.  :)

**Comment by maxckelly** [comment]
Thank you! :)

**Pull request closed without merging**